### DISCUSSION TITLE
How to continuously play the voice of all players at the corresponding time in the demo？

### DISCUSSION THREAD

**Discussion by Lelukenian** [akiver/cs-demo-manager#800] [Ideas]
In the demo, listen to everyone's voice, whether it's from CT or T, and learn other people's tactics. You can also hear the praise of opponents and teammates during consecutive kills. Can a function be implemented during demo playback to continuously play all voice communication situations

**Comment by TreyPorter** [comment]
This is fairly easy to set up with a few console commands:
```
tv_listen_voice_indices -1
tv_listen_voice_indices_h -1
```
To set this up as the default whenever you launch CS2

1. If it doesn't exist already. in `SteamLibrary\steamapps\common\Counter-Strike Global Offensive\game\csgo\cfg` create `autoexec.cfg`
2. Add the 2 lines above to `autoexec.cfg`

If you only want this to be the default when using CS Demo Manager

1. Create a new cfg file in `SteamLibrary\steamapps\common\Counter-Strike Global Offensive\game\csgo\cfg`, such as `demoComms.cfg`
2. Add the 2 lines above to your new cfg file 
3. Navigate to CS Demo Manager > Settings > Playback
4. Set `Launch Parameters` to `+exec demoComms.cfg`

**Comment by akiver** [comment]
Hi!
Since v3.6.0 there is an option to enable players' voice (works only if the demo contains voice data which is not the case with MM demos).
